### DISCUSSION TITLE
Product with multiple productFamilty

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#613] [Q&A]
In filling productFamilies with attributes, we discovered that our Product have to have only single product family
Is there any work around this problem, so we can assign multiple product families to product?
![image](https://user-images.githubusercontent.com/30953478/216512488-4c2f7fe2-c479-4d3b-84ec-33862c3fdef8.png)

Thank you for your support!

**Comment by zssvaidar** [comment]
We can't have same attribute in multiple attributeGroup, also Products can have only single productFamily
we know that by separating attributes to groups then adding them to family is design choice, but in our case we have to have product attributes of one class and class below to be separate

**Reply by atrolex** [reply]
Having attribute assigned only to a single attribute group is our "design" decision. If you want to assign an attribute to multiple groups consider duplicating this attribute, you can also give it the same name, but it's code should be unique.

**Comment by atrolex** [comment]
We work now on improving our paid module "advanced classification". The possibility to assign multiple product families to a product will be added in this module.
